promql/bench_test.go

```diff
@@ -66,10 +67,16 @@ func setupRangeQueryTestData(stor *teststorage.TestStorage, _ *Engine, interval,
 			ref, _ := a.Append(refs[i], metric, ts, float64(s)+float64(i)/float64(len(metrics)))
 			refs[i] = ref
 		}
+		metric := labels.FromStrings("__name__", "sparse", "l", strconv.Itoa(s/(numIntervals/50)))
```

Comment:
Suggest a comment here saying why `sparse` exists, why it is different from `a_ten` etc.
(Maybe also a comment briefly explaining `a_ten` etc. is also a good idea)

Reply:
PTAL?

Reply:
Much better.  I think it can be more succinct, and also more informative:

```
// Generating a sparse time series: each label value of "l" will contain data only for
// one in 50 points in the time range.
```

(I don't know if this is correct, I find it very confusing to read.  Why is the index variable called `s`?  Maybe splitting your addition out from the loop that calls `Commit` would improve readability.)

Reply:
Hum.. The intent here was to create 50 timeseries...

So i divided the numIntervals/50 and then s by the result:

So lets say i have:

numIntervals=2000
pointsPerSparseSeries = 2000/50 = 40
 
So now if i divided s/40 i will have l=0 for the first 40 points, s = 1 for the next 40 ones, so on so forth.

Let me try to rewrite the code and comment.